pkg/services/ngalert/CHANGELOG.md

```diff
@@ -50,5 +50,6 @@ Scopes must have an order to ensure consistency and ease of search, this helps u
 - [BUGFIX] Scheduler: Fix state manager to support OK option of `AlertRule.ExecErrState` #47670 
 - [ENHANCEMENT] Templates: Enable the use of classic condition values in templates #46971
 - [ENHANCEMENT] Scheduler: ticker expose new metrics `grafana_alerting_ticker_last_consumed_tick_timestamp_seconds`, `grafana_alerting_ticker_next_tick_timestamp_seconds`, `grafana_alerting_ticker_interval_seconds` #47828 
+- [ENHANCEMENT] Scheduler: ticker to support stopping #48142
```

Comment:
Does this need a changelog entry? AFAICT it doesn't affect a customer's usage of Grafana or fix anything that's broken.

Reply:
Good point. This changelog is kinda internal and does not mention the fact that the change must be user-facing.